src/presets.js

```diff
@@ -85,7 +86,8 @@ const templateDefaults = [
 
 	'{{#*inline "block-newline"}}',
 		'{{#isnt block ""}}{{block}} {{else}}{{/isnt}}',
-	'{{/inline}}'
+	'{{/inline}}',
+	''
```

Comment:
This was added because it removes a bunch of new lines that I don't want added. I'm not sure why it works just that it does precisely what I need and doesn't seem like it should be harmful?